pkg/status/status.go

```diff
@@ -356,11 +357,12 @@ func (s *status) monitorClusterStatus() {
 			// Create the ClusterOperator in the progressing state if it does not exist
 			// or if it is the first report.
 			if s.clusterOperator == nil {
+				reason := "operatorProgressing"
```

Comment:
Can't you specify the version you are progressing towards here?

Reply:
We can add the version if we believe it adds value.

I believed that it did not add value after looking at what information is available within telemetry. Telemetry provides the current version and the target version that the cluster is progressing to - so I thought repeating this information in the reason was a no-value-add. If you disagree, we can update this message to reflect the version, such as `OperatorProgressingTo4.2.0-0.ci-2019-07-10-180724`.

Reply:
Given that telemetry provides the info, I am fine leaving it out. If **possible** please provide more context in each case.